.github/workflows/periodic.yml

```diff
@@ -0,0 +1,98 @@
+# Workflow intended to periodically test containerd integration tests
+
+name: win-periodic-integration
+
+on:
+  workflow_dispatch:
+
+env:
+  AZURE_RESOURCE_GROUP: 'adtv-temp-${{ github.run_id }}'
+  AZURE_DEFAULT_LOCATION: westeurope
+  AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUB_ID }}
+  AZURE_DEFAULT_VM_SIZE: Standard_D2s_v3
+  SAC_1903_IMG: "MicrosoftWindowsServer:WindowsServer:Datacenter-Core-1903-with-Containers-smalldisk:18362.1198.2011031735"
+  PASSWORD: Passw0rdAdmin # temp for testing, will be generated
+  DEFAULT_ADMIN_USERNAME: azureuser
+  SSH_OPTS: "-o ServerAliveInterval=20 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+  REMOTE_VM_BIN_PATH: "c:\\containerd\\bin"
+
+jobs:
+  win1903:
```

Comment:
How long would it take? Would it make sense to run the tests on PRs as well?

Reply:
From start to finish, including cri-integration tests ( enabled by a separate PR that enables them ) would be something like maximum 50 minutes. That includes artefact uploading and machine decommissioning. It would eventually, but that is not the goal for the short term. Also there are budgetary challenges to consider before migrating this to a PR presubmit test.